homeassistant/components/tcp/binary_sensor.py

```diff
@@ -1,22 +1,31 @@
 """Provides a binary sensor which gets its values from a TCP socket."""
+from __future__ import annotations
+
 from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
-from .sensor import CONF_VALUE_ON, PLATFORM_SCHEMA, TcpSensor
+from .const import CONF_VALUE_ON
+from .sensor import PLATFORM_SCHEMA, TcpSensor
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({})
```

Comment:
Please fix this one as well :)

Reply:
not sure, but should not a simple `PLATFORM_SCHEMA: Final = TCP_PLATFORM_SCHEMA`  do the trick, so no `extend()` or copy is needed :thinking:

Reply:
Yes, that's what I meant by "simple reference should be fine".

Reply:
ups ... i have overseen the second sentence  :see_no_evil: